hack/make/prerequisites.mk

```diff
@@ -16,6 +16,10 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+## Install all prerequisites
+prerequisites:
+	./hack/prerequisites/run_all.sh
```

Comment:
you can do it without hack shell.

We already have `make deps` to run all:
```
## Installs dependencies
deps: prerequisites/setup-pre-commit prerequisites/kustomize prerequisites/controller-gen
```

Reply:
Nice that you noticed :+1: 
I replaced `make deps` with `make prerequisites` as it makes more sense to keep the same naming. Actually that's the reason why I didn't notice about the command, because it has a different name!